module/commands/welcome.py

```diff
@@ -23,6 +23,14 @@ def generate_welcome(new_member: User) -> str:
 
 
 def send_welcome(update: Update, _: CallbackContext) -> None:
-    for new_member in update['message']['new_chat_members']:
-        if not new_member['is_bot']:
-            update.message.reply_text(f'{generate_welcome(new_member)}')
+    if update.message.new_chat_members:
+        for new_member in update.message.new_chat_members:
+            if not new_member.is_bot:
+                handle_welcome(update, new_member)
+
+
+def handle_welcome(update: Update, new_member: User) -> None:
+        welcome_msg = f'{generate_welcome(new_member)}\n' + \
+                      f"- Dai un'occhiata al [README]({welcome['readme']})\n" + \
+                      f"- {welcome['utils'][randrange(0, len(welcome['utils']))]}"
```

Comment:
Facendo riferimento al commento su `utils`, andrebbero localizzate le stringhe, eventualmente memorizzando il valore per l'uso successivo

Reply:
E come potrebbe diventare "randomico" il messaggio se appena ne leggo uno casuale dal file, esso viene localizzato e ripreso in una futura chiamata della funzione?

Reply:
Errore mio, intendevo memorizzare la lingua di riferimento in modo da essere passata all'interno di `generate_welcome`, senza dover quindi ottenere di nuovo il valore da `User`

Reply:
Quindi l'idea sarebbe creare una sorta di:

```json
{
     "it" : {
       "benvenuti" : ["...", "..." ],
       "utils" : ["...", "...", "..." ]
      }
      "readme" : "https://..."
}

```
Ho capito bene?

Ed eventualmente la il `lan_code` conservato in una variabile

Reply:
Esattamente c: